measure_iops/src/main.rs

```diff
@@ -191,7 +199,7 @@ async fn main() -> Result<()> {
 
             io_operations_sent = 0;
             bw_consumed = 0;
-            io_operation_time = Instant::now();
+            measurement_time = Instant::now();
```

Comment:
Note that you also need to clear `total_io_time` here to make sure that `fractional_seconds` doesn't measure the I/O time since the start of the run. e.g.:

```suggestion
            measurement_time = Instant::now();
            total_io_time = Duration::ZERO;
```